doc/engineering-reference/src/advanced-surface-concepts/exterior-naturally-vented-cavity.tex

```diff
@@ -124,18 +134,7 @@ \subsection{Cavity Heat Balance}\label{cavity-heat-balance}
 
 \(\rlap{--} \dot{V}_{tot} = \rlap{--} \dot{V}_{wind} + \rlap{--} \dot{V}_{thermal}\) ~is the total volumetric flow rate of air ventilating in and out of the cavity.
 
-\begin{equation}
-\rlap{--} \dot{V}_{wind} = C_{v}A_{in}U_{\infty}
-\end{equation}
-
-\begin{equation}
```

Comment:
This equation is gone now?

Reply:
@Myoldmopar Hmmm...might be a bad edit there.  I will need to go back and take a look at this, but I would like to finish up the third set of stuff first (hopefully that will happen by tomorrow--things have been a little slower than anticipated but I'm still trying to get through the whole thing before the release).  I'll get back to you on this...

Reply:
OK, let me know when you get a chance to look at it.  Since you don't seem to recall it being intentional, I kinda feel like it just needs to be put back in.  I'm happy to drop it back in right away to help get these PRs back in, just let me know if you concur.

Reply:
I took another look and I think this is OK.  The thermal equations are still in there, but Git(hub) is struggling to show it.  I think we're good here.

Reply:
@Myoldmopar Oh good.  I wonder if due to the shear number of changes that Git(hub) is struggling?

Reply:
Either way I still see those equations in there in my local branch where I took develop and merged this and the second round of changes.  So I'm happy.  I'm going to merge them both now.